stories/internal/DualAxesChartConfig.tsx

```diff
@@ -0,0 +1,494 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { screenshotWrap } from '@gooddata/test-storybook';
+import * as dataSet from '../test_data/chart_with_2_metrics_and_view_by_attribute';
+import { Visualization } from '../../src/components/visualizations/Visualization';
+import { wrap } from '../utils/wrap';
+import '../../styles/scss/charts.scss';
+
+storiesOf('Internal/DualAxesMinMaxConfig', module)
+    .add('Dataset with 0+ data on both axes', () => {
+        return screenshotWrap(
+            wrap(
+                <div>
+                    <div>Left: Min(not set), Max(not set)</div>
```

Comment:
What about some wrapper component which will render this heading?

Reply:
I add MinMaxInfo component to wrap the min/max info